contributing/workflow.md

```diff
@@ -363,28 +363,16 @@ Update your `4.3` branch:
 > git push origin 4.3
 ```
 
-Create a new branch `feat-ab.new` from the correct branch `4.3`:
-
-```console
-> git switch -c feat-abc.new 4.3
-```
-
-Cherry-pick the commits you did:
-
-```console
-> git cherry-pick <commit_id> <commit_id> <commit_id> ...
-```
-
-Rename the PR branch `feat-abc`:
+(Optional) Create a new branch as a backup, just in case:
 
 ```console
-> git branch -m feat-abc feat-abc.old
+> git branch feat-abc.bk feat-abc
```

Comment:
Shouldn't this use `-c`?

Reply:
I don't intend to switch to the new branch.

Reply:
I didn't realize you could create a new branch like that! I've switched to `switch` for a while now so I'm rusty on all the `branch` commands other than listing.